### PR TITLE
refactor(Communities): replace `CommunityMembersPopup` with `Communit…

### LINKED ISSUE #2890: Replace `CommunityMembersPopup` with `StatusModal`

### PR DESCRIPTION
**Pull request by PascalPrecht**
…yProfilePopup`

When right-clicking a community from the navbar and selecting "View community", Status Desktop
opens `CommunityMembersPopup` which looks like it was either never really finished or more of an
intermediate solution until a proper community profile popup was created.

That's why this commit replaces it with a `CommunityProfilePopup` instead.
In fact, this lead to changes in `CommunityProfilePopup` where the `activeCommunity` dependency
is entirely removed, which allows us to use this popup in various places given that it's hydrated
with proper data.

Because we're no longer relying on `activeCommunity` inside that popup, all of its children and
connected popups needed that same refactor as well, hence this PR introduces a few more changes.

Closes #2890

### CODE REVIEW

File: ui/app/AppLayouts/Chat/CommunityColumn.qml [hunk 1]
```diff
@@ -35,7 +35,9 @@ Item {
             qsTr("%1 Members").arg(chatsModel.communities.activeCommunity.nbMembers)
         chatInfoButton.image.source: chatsModel.communities.activeCommunity.thumbnailImage
         chatInfoButton.icon.color: chatsModel.communities.activeCommunity.communityColor
-        chatInfoButton.onClicked: communityProfilePopup.open()
+        chatInfoButton.onClicked: openPopup(communityProfilePopup, {
+            community: chatsModel.communities.activeCommunity
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/CommunityColumn.qml:39]: FYI, from what I learned from @emizzle , doing something like this doesn't bind the data, so if something changes in `activeCommunity`, the popup's data wouldn't update. Is that a problem? If the popup is meant only for info, then close, I feel like it's ok.

**Review by jrainville — COMMENTED** [review]
Looks good. I didn't approve yet to get your feedback on the data binding question

File: ui/app/AppLayouts/Chat/CommunityColumn.qml (hunk 1 above)
Comment (PascalPrecht) [reply ui/app/AppLayouts/Chat/CommunityColumn.qml:39]: Ah, that's a good point.. I guess if we want to be super sure, we should go the safe route.

On the other hand, how much can possibly change in the community when this popup was opened? Maybe its description, image and members, but I think it's fine if that isn't updated in realtime while the popup is open.

Happy to change tho, of there's strong opinions!

Comment (jrainville) [reply ui/app/AppLayouts/Chat/CommunityColumn.qml:39]: Yeah, I don't think it should  update much, so I think for now it's fine as is.

**Review by jrainville — APPROVED** [review]

**Pull request merged**